core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -698,32 +716,32 @@ class EMConfig {
 
     @Experimental
     @Cfg("Specify a probability to apply S1iR when resource sampling strategy is 'Customized'")
-    @Min(0.0)@Max(1.0)
+    @Probability
     var S1iR : Double = 0.25
 
     @Experimental
     @Cfg("Specify a probability to apply S1dR when resource sampling strategy is 'Customized'")
-    @Min(0.0)@Max(1.0)
+    @Probability
     var S1dR : Double = 0.25
 
     @Experimental
     @Cfg("Specify a probability to apply S2dR when resource sampling strategy is 'Customized'")
-    @Min(0.0)@Max(1.0)
+    @Probability
     var S2dR : Double = 0.25
 
     @Experimental
     @Cfg("Specify a probability to apply SMdR when resource sampling strategy is 'Customized'")
-    @Min(0.0)@Max(1.0)
+    @Probability
     var SMdR : Double = 0.25
 
     @Experimental
     @Cfg("Specify a probability to enable archive-based mutation")
-    @Min(0.0) @Max(1.0)
+    @Probability
     var probOfArchiveMutation = 0.0
 
     @Experimental
     @Cfg("Specify a percentage which is used by archived-based gene selection method (e.g., APPROACH_GOOD) for selecting top percent of genes as potential candidates to mutate")
-    @Min(0.0) @Max(1.0)
+    @Probability
```

Comment:
thanks, Andrea. I am not sure whether we change this to `Probability` since this indicates a percentage. If we decide to change it, we also need to change `focusedSearchActivationTime`.

Reply:
good point! thx. i ll make a fix

Reply:
just pushed a fix